tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3784,20 +3815,52 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        isSameInferenceClass_: function(val1, val2) {
+          return this.isRegression_(this.modelType)
+            ? Math.abs(val1 - val2) < this.minCounterfactualValueDist
+            : val1 == val2;
+        },
+
+        adjustMaxCounterfactualValueDist_: function(selected, valueStr) {
+          this.maxCounterfactualValueDist = Math.max(
+            this.distanceStats_[valueStr].max -
+              this.visdata[selected][valueStr],
+            this.visdata[selected][valueStr] - this.distanceStats_[valueStr].min
+          );
+        },
+
+        adjustMinCounterfactualValueDist_: function() {
+          const valueStr = this.strWithModelName_(
```

Comment:
`valueStr` -> `modelName` seems more appropriate.

Reply:
It's the name of the key holding the value, which is basically the string `' Inference value'` suffixed with the model name... I've changed it to `valueName` to make it clearer.